patches/tModLoader/Terraria/release_extras/DedicatedServerUtils/README.md

```diff
@@ -0,0 +1,65 @@
+# Dedicated Server Utils
+This directory contains utilities for managing a dedicated ***Linux*** server.
+
+## Docker vs Management Script
+Both the Docker container and the management script can install and update tModLoader and any mods. Docker is a service used to containerize, or isolate, a specific application from a host system. This is beneficial for both security and ease of use with more complicated services. If you would prefer more control over your server, then make use of `manage-tModLoaderServer.sh`.
+
+## Using The Script
+The `manage-tModLoaderServer.sh` script can be used to install tModLoader either directly from the GitHub release or from SteamCMD. The script is made to run fully standalone, so just download it to your server and run it. No other files from the repo are needed.
+
+### Installing tModLoader
+#### Via SteamCMD (recommended)
+* Ensure SteamCMD is installed and on your PATH. You can install SteamCMD from your package manager or [Valve's Wiki](https://developer.valvesoftware.com/wiki/SteamCMD).
+* Installation via SteamCMD can be performed using `./manage-tModLoaderServer.sh --install --username your_steam_username`.
+* You will be prompted for your password (and your 2fa code if applicable).
+* By default, tModLoader will be installed to `~/Steam/steamapps/common/tModLoader`. To specify an installation directory, use `--folder /full/path/from/root`
+
+#### From GitHub
+* Installation from the latest GitHub release can be performed using `./manage-tModLoaderServer.sh --install --github`.
+* By default, tModLoader will be installed to `~/tModLoader`. To specify an installation directory, use `--folder /path/to/install`.
+
+Additionally, you can avoid updating or installing mods with the `--no-mods` argument.
+
+### Installing Mods
+Mods will be automatically installed during the tModLoader installation step, but can also be installed separately using the `--mods-only` argument. Simply place any `.tmod` files, `install.txt` for workshop mods, and `enabled.json` into the same directory as the script.
+
+#### Obtaining install.txt
+Because the steam workshop does not use mod names to identify mods, you must create a modpack to install mods from the workshop. To get an `install.txt` file and its accompanying `enabled.json`:
+* Go to Workshop
+* Go to Mod Packs
+* Click `Save Enabled as New Mod Pack`
+* Click `Open Mod Pack Folder`.
+* Enter the folder with the name of your modpack
+
+You can copy `enabled.json` and `install.txt` to your script directory and they will be used next time the script is run.
+
```

Comment:
What command do I need to run to install mods? Maybe remind people to run `manage-tModLoaderServer.sh --mods-only` after adding their mods/install.txt to the folder?